core/src/main/java/org/fao/geonet/kernel/SchemaManager.java

```diff
@@ -206,13 +208,14 @@ private void addResolverRewriteDirectives(GeonetworkDataDirectory dataDir) {
      * @param defaultLang      the default language (taken from context)
      * @param defaultSchema    the default schema (taken from config.xml)
      */
-    public void configure(ApplicationContext applicationContext, Path basePath, Path resourcePath, Path schemaPluginsCat,
+    public void configure(ApplicationContext applicationContext, Path basePath, Path resourcePath, Path schemaPublicationDir, Path schemaPluginsCat,
```

Comment:
Please update the documentation of the parameters, we have now, several paths and it's difficult to have clear what means each one:

- `resourcePath`
- `schemaPublicationDir`
- `schemaPluginsCat`

Reply:
Updated.